control/execute/control_run.go

```diff
@@ -96,7 +96,7 @@ func (r *ControlRun) Start(ctx context.Context, client *db.Client) {
 	}
 
 	// set a log line in the database logs for convenience
-	_, _ = client.ExecuteSync(ctx, fmt.Sprintf("--- Executing %s", *control.Title))
+	_, _ = client.ExecuteSync(ctx, fmt.Sprintf("--- Executing %s", *control.Title), false)
```

Comment:
This can be set to `true`. This is used to set a comment in the database log for convernience